docker-compose.yml

```diff
@@ -1,32 +1,35 @@
-version: '2'
+version: '3'
+
 services:
-    image: sabaitech/seafile:latest
-    container_name: seafile
+  seafile:
+    build: .
     ports:
      - "8000:8000"
      - "8082:8082"
+    # connect localhost ports for oauth2 callback
+    network_mode: "host"
     environment:
-     - SERVER_ADDRESS=my.domain.com
-     - FASTCGI=true
-     - MYSQL_SERVER=mariadb
-     - MYSQL_USER=$SEAFILE_USER
-     - MYSQL_USER_PASSWORD=$SEAFILE_PASSWORD
-     - MYSQL_ROOT_PASSWORD=$MARIADB_ROOT_PASSWORD
-     - SEAFILE_ADMIN=$SEAFILE_ADMIN
-     - SEAFILE_ADMIN_PASSWORD=$SEAFILE_ADMIN_PASSWORD
+      SERVER_ADDRESS: $SEAFILE_SERVER_ADDRESS
```

Comment:
In the same way, as it is an example file it is more understandable to use explicit value instead of variables.